Framework/DataHandling/src/PDLoadCharacterizations.cpp

```diff
@@ -540,22 +541,29 @@ void PDLoadCharacterizations::readVersion1(const std::string &filename,
       if (result.size() == 2) {
         line = Strings::strip(result[1]);
         Kernel::StringTokenizer tokenizer(
-            line, " ", Kernel::StringTokenizer::TOK_IGNORE_EMPTY);
+            line, " \t", Kernel::StringTokenizer::TOK_IGNORE_EMPTY);
         for (const auto &token : tokenizer) {
           columnNames.push_back(token);
         }
       }
     } else {
       if (columnNames.empty()) // should never happen
-        throw std::runtime_error("file missing column names");
+        throw Exception::FileError("file missing column names", filename);
 
       line = Strings::strip(line);
       Kernel::StringTokenizer tokenizer(
-          line, " ", Kernel::StringTokenizer::TOK_IGNORE_EMPTY);
+          line, " \t", Kernel::StringTokenizer::TOK_IGNORE_EMPTY);
       std::vector<std::string> valuesAsStr;
       for (const auto &token : tokenizer) {
         valuesAsStr.push_back(token);
       }
+      if (valuesAsStr.size() < columnNames.size() + 6) {
```

Comment:
What's the number six here? Could it be put in a `constexpr` with some descriptive name?

Reply:
I'll make a constant with a descriptive name for the number 6 (also appears elsewhere for the same reason). `constexpr` doesn't apply because you don't know the value of `columnNames.size()` at compile time.